sequencer/src/lib.rs

```diff
@@ -140,6 +140,11 @@ impl<N: network::Type> NodeImplementation<SeqTypes> for Node<N> {
     type CommitteeNetwork = N::DAChannel;
 }
 
+#[derive(Clone, Copy, Debug, Default)]
+pub struct Delta {}
```

Comment:
We can leave this empty for now, added this issue: 

https://github.com/EspressoSystems/espresso-sequencer/issues/1242

Reply:
Sorry, Do you mean `pub struct Delta ()`?

Reply:
Moreover, the associated type also has a trait bound. So thought of this doing this way

Reply:
I mean it's fine like this but I assumed at some point we want to be more than a struct with no members. I meant to say we can do it in another PR.

Reply:
Oh I see, I thought you want some changes for it. Cool!